src/table/Root.jsx

```diff
@@ -1,10 +1,14 @@
 import ReactDOM from 'react-dom';
 import React from 'react';
 import { ThemeProvider, StylesProvider } from '@material-ui/core/styles';
+import muiSetup from '../mui-setup';
 import TableWrapper from './TableWrapper';
 
+// This line is replaced by rollup with an import for internal builds
+const __OPIONAL_THEME_DEPS__ = {}; // eslint-disable-line no-underscore-dangle
+
 export function render(rootElement, props) {
-  const { muiParameters } = props;
+  const muiParameters = muiSetup(__OPIONAL_THEME_DEPS__);
```

Comment:
Why is this faster? Every time `render` runs , we have to run `muiSetup`?

Reply:
component runs more often than render. when we put muiSetup() in component, it runs more often.

Reply:
well, it is not supposed to. Since we are using `useState` and using `muiSetup` as the initial value, it should just run once?

Reply:
no, but we can use a function to lazily initialize the variable,  it will be executed only on the initial render. I will fix that.

Reply:
Yupp that would do the job :)